docs/installation.md

```diff
@@ -216,13 +215,14 @@ with real users.
 | BOOTDELAY | All | How long to wait for terraform deployed instances to boot before continuing with install, in minutes |
 | SKIP_SF_TEST | All | Set to 1 to skip running destructive testing of the cloud |
 | KSM_ENABLED | All | Set to 1 to enable KSM, 0 to disable |
+| NODE_COUNT | gcp | Number of SF hypervisor nodes to deploy |
```

Comment:
This should be down with the other GCP specific options unless we're going to add it to AWS etc.